crates/cli/src/lib.rs

```diff
@@ -1145,7 +1169,7 @@ fn roc_run_native<I: IntoIterator<Item = S>, S: AsRef<OsStr>>(
     use bumpalo::collections::CollectIn;
 
     let executable = roc_run_executable_file_path(binary_bytes)?;
-    let (argv_cstrings, envp_cstrings) = make_argv_envp(arena, &executable, args);
+    let (argv_cstrings, envp_cstrings) = make_argv_envp(arena, script_path, args);
```

Comment:
From the added tests, it looks like a script in `/home/myuser/Documents/myscript.roc` would have the relative path included along with the name, but I think we should prefer just the [file_name](https://doc.rust-lang.org/std/path/struct.Path.html#method.file_name) (a.k.a. `myscript.roc`) since that's what will be wanted to display 99% of the time.

Reply:
I think the relative name is important if we want to use argv[0] to look up the script location in the application.

For instance, if I'm in /home/jasper and run: `roc ./projects/foobar/main.roc`, if argv[0] only tells me I'm running `main.roc`, then in the implementation of `main.roc` I can't know where the script is located, for instance to read some `data.json` file that's supposed to be next to the script file.

Reply:
Okay, if we're doing what bash is doing, I'm happy.